CMakeLists.txt

```diff
@@ -10125,6 +10125,258 @@ if(gRPC_BUILD_TESTS)
 
 add_executable(alts_grpc_record_protocol_test
   test/core/tsi/alts/zero_copy_frame_protector/alts_grpc_record_protocol_test.cc
+  src/core/ext/filters/client_channel/resolver/fake/fake_resolver.cc
```

Comment:
Is this change intentional?

Reply:
Yes, this change showed up because it needs to have new dependency against `grpc_test_util_base` to use `grpc::testing::TestEnvironment`